### ISSUE TITLE
Document Model Training

### ISSUE THREAD

**Issue by SubjectiveScientist** [Olament/DeepMushroom-docker#1]
How is the PyTorch model trained? What parameters are used (especially regularization)?

The README suggests that the DeepMushroom repo is used, however that is fast.ai- rather than pytorch-based, and the export to file is not clear from the other repo.

**Comment by Olament** [comment]
Sorry for the confusion. I have not pushed my actual PyTorch model to the Github yet. I am still experimenting with new parameters and thats why the model is not there. I will upload the code once I finished experiment but I do not have an ETA for that. 

For regularization techniques, I have dropout (with p=0.5) at my FC layer, which I think is bascially the default setting for fully connected layer. Except that, I do not use other regularization techniques. From my personal experience, it is hard to overfit DCNNs and my model is sightly underfitting.
